### PR TITLE
CORE-3776 - Buttons result in a 404 error in the order history view when accessed from the client card

### PR DESCRIPTION
**Pull request by abdyfranco**

### CODE REVIEW

**Comment by JReissmueller** [comment]
Update the version to 1.0.1

File: client_cards_plugin.php [hunk 1]
```diff
@@ -18,6 +18,8 @@ public function __construct()
         Language::loadLang('client_cards_plugin', null, dirname(__FILE__) . DS . 'language' . DS);
 
         $this->loadConfig(dirname(__FILE__) . DS . 'config.json');
+
+        $this->client_uri = WEBDIR . Configure::get('Route.client') . '/';
```
Comment (JReissmueller) [inline client_cards_plugin.php]: You test this?  Adding WEBDIR here breaks it for me.  Pretty sure that part is already taken care of by the public_uri appended in the core

Comment (abdyfranco) [reply client_cards_plugin.php]: Yes, I tested it and worked for me. But yes, you're right, public_uri takes care of the web dir. Updated.

Comment (JReissmueller) [reply client_cards_plugin.php]: Got it, it's probably because my installation is in a sub directory

**Review by JReissmueller — APPROVED** [review]

**Pull request merged**